src/interactions/interaction.ts

```diff
@@ -24,17 +25,22 @@ export class Interaction {
    * @returns {Interaction} The calling Interaction.
    */
   public attachTo(component: Component) {
-    if (this._componentAttachedTo) {
-      this.detachFrom(this._componentAttachedTo);
-    }
-
+    this._detach();
     this._componentAttachedTo = component;
-    component.onAnchor(this._anchorCallback);
-
+    this._attach();
     return this;
   }
 
   /**
+   * Attaches to existing _componentAttachedTo if enabled.
```

Comment:
The docs point to a private instance variable which sounds.. a bit incorrect.  Javascript users now might wonder what this `_componentAttachedTo` is and then run into trouble interacting with it.

Let's just use english here.  This also helps us if we decide to change the name of the variable later down the road